### DISCUSSION TITLE
Issues to external sources

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#5041] [General]
This discussion is intended to aggregate tickets that we make for external sources.

## Disease Ontology
- https://github.com/DiseaseOntology/HumanDiseaseOntology/issues/1045

**Comment by nicolevasilevsky** [comment]
I emailed Orphanet about the mapping between Orphanet:95494 Combined pituitary hormone deficiencies, genetic forms and OMIM:613038 PITUITARY HORMONE DEFICIENCY, COMBINED OR ISOLATED, 1; CPHD1.
See:
https://github.com/monarch-initiative/mondo/issues/4741#issuecomment-1167990062
https://github.com/monarch-initiative/mondo/issues/4741#issuecomment-1169142506

**Comment by matentzn** [comment]
[ORDO download URLs](https://github.com/Orphanet/ORDO/issues/35)

**Comment by matentzn** [comment]
https://github.com/Orphanet/ORDO/issues/37
